### ISSUE TITLE
Turn generation field in fatality into an Enum

### ISSUE THREAD

**Issue by rgreinho** [scrapd/scrapd#211]
Issue Type
----------

- Feature request

Current Behavior
----------------

The generation field in a fatality is a free form text field.

Expected Behavior
-----------------

The generations are a set of known values, therefore could be structured as an Enum in the `Fatality` model to add more validation.

Possible Solution
-----------------

The Enum must contain the known generations. If a generation is unknown, it should default to blank.

**Comment by lalver1** [comment]
I implemented this change and it seems to be working. I'll try to submit a PR soon.

**Comment by rgreinho** [comment]
It should be OK, but we will also need to validate that this change in the model does not break the pipeline. If it does, the pipeline tools will need to be upgraded as well.

**Comment by lalver1** [comment]
That makes sense. Should I submit the PR? I'm not sure if CircleCI will validate the changes or if I need to run tests locally before I submit.

**Comment by rgreinho** [comment]
Yes, absolutely! Submit it! You did all the work. You'll just have to provide a way to manually verify that it does not break the pipeline.

You can get some inspiration from the [ScrAPD 2 to 3 migration script](https://github.com/scrapd/datasets/wiki/ScrAPD-2-to-3-migration) we're going to run.
